examples/kubernetes/snapshot/specs/snapshot-import/volume-snapshot-content.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: snapshot.storage.k8s.io/v1alpha1
+kind: VolumeSnapshotContent
+metadata:
+  name: static-snapshot-content
+spec:
+  csiVolumeSnapshotSource:
+    driver: ebs.csi.aws.com 
+    snapshotHandle: snap-0fba4d7649d765c50
```

Comment:
Do you want to leave this handle here just as an example? Or would you rather leave it empty?

Reply:
I would prefer to leave the handle as an example so that people know the format of it in case they put in some other string (maybe snapshot name) and it fails to work.